package.json

```diff
@@ -55,6 +55,7 @@
     "@types/react": "^18.0.26",
     "@types/react-native": "^0.70.0",
     "@types/react-test-renderer": "^18.0.0",
+    "@types/styled-components": "^5.1.26",
```

Comment:
I think we also need `@types/styled-components-react-native`
https://styled-components.com/docs/api#typescript

Reply:
updated